src/Compilers/CSharp/Test/Emit/PDB/PDBTests.cs

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
I think it's the result of the separator I picked for formatting (║). I don't see any harm to this distinct character, but I'm open to suggestions.